web-rdfs/source/web_request_translator.cc

```diff
@@ -129,7 +129,7 @@ namespace webRequestTranslator {
 
     for (i = 0; i < num_files; i++) {
       if (i > 0) {
```

Comment:
Shouldn't this be `>=`?

Reply:
no because if there is only one file you will have an extra comma